src/DateTime/tests/test_datetime.py

```diff
@@ -228,12 +230,26 @@ def test_compare_methods(self):
         # Compare a date to int
         dt = DateTime(1)
         self.assertEqual(dt, DateTime(1.0))
+        self.assertTrue(dt == DateTime(1))  # testing __eq__
+        self.assertFalse(dt != DateTime(1))  # testing __ne__
         self.assertFalse(dt.greaterThan(1))
         self.assertTrue(dt.greaterThanEqualTo(1))
         self.assertFalse(dt.lessThan(1))
         self.assertTrue(dt.lessThanEqualTo(1))
         self.assertFalse(dt.notEqualTo(1))
         self.assertTrue(dt.equalTo(1))
+        # Compare a date to string; there is no implicit type conversion
+        # but behavior if consistent as when comparing, for example, an int
+        # and a string.
+        dt = DateTime("2023")
+        self.assertFalse(dt == "2023")  # testing __eq__
```

Comment:
@icemac the idea with the `# testing __eq__` code comment here is in case one day we use a linter or something that suggests rewriting using the natural  `self.assertNotEqual(dt, "2023"), but this would be incorrect here, because `assertNotEqual` checks `!=` and this is a test for `__eq__`.

Writing this I feel that maybe this line could be written as

```py
        self.assertFalse(dt.__eq__("2023"))
```

Reply:
ok, then no objection for your changes here from me. 😉

Reply:
Thanks, then if everybody is OK with these changes, somebody has to approve then I will merge.